src/autoscaler/scalingengine/fakes/fake_scalingengine_db.go

```diff
@@ -16,12 +16,13 @@ type FakeScalingEngineDB struct {
 	saveScalingHistoryReturns struct {
 		result1 error
 	}
-	RetrieveScalingHistoriesStub        func(appId string, start int64, end int64) ([]*models.AppScalingHistory, error)
+	RetrieveScalingHistoriesStub        func(appId string, start int64, end int64, orderType db.OrderType) ([]*models.AppScalingHistory, error)
```

Comment:
Yes that what I wanted to ask , was not manually modified by you but generated using Counterfeiter . Thanks!